### ISSUE TITLE
Rename routes.py to views.py

### ISSUE THREAD

**Issue by ModoUnreal** [ModoUnreal/nuncio#4]
Self-explanatory really, just make sure that you change the import statements aswell.

**Comment by ns23** [comment]
Can I work on this issue ??

**Comment by ModoUnreal** [comment]
Yeah sure! Remember to rename the imports to views as well, so instead of `import app.routes` you change it to `import app.views`.
